### ISSUE TITLE
Respect files property from karma.config on `tns test --bundle`

### ISSUE THREAD

**Issue by Fatme** [NativeScript/nativescript-cli#4509]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 
 - Cross-platform modules:
 - Android Runtime:
 - iOS Runtime:
 - Plugin(s):

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Sample project**

**Additional context**

**Comment by Fedelaus** [comment]
Is this still an issue?

**Comment by Fatme** [comment]
@Fedelaus,

Yes, it is still an issue. Currently `test` command respects only files from `tests` folder.

**Comment by poppahorse** [comment]
@Fatme @rosen-vladimirov can we get an update on this and why its been removed from the `6.0.0` release please?

This is something really important to our organisations use of NS and as @Fedelaus says there's been a lot of obfuscation and struggle around unit tests for a while.
